osu.Game/Screens/Edit/Components/Timelines/Summary/Parts/KiaiPart.cs

```diff
@@ -91,7 +92,8 @@ public KiaiSection Section
                     section = value;
 
                     X = (float)value.StartTime;
-                    Width = (float)value.Duration;
+                    // Minimum width ensures that very short kiai sections still show a slither of colour.
+                    Width = (float)Math.Max(200, value.Duration);
```

Comment:
This only kinda works when the track is of "normal" length and not a 30-minute marathon. And the "kinda" looks like this:

![Screenshot 2025-06-24 at 09 44 03](https://github.com/user-attachments/assets/c56426a7-b581-42ed-9509-f12c81218fb1)

Sure you're ok going with this as an attempt to fix the issue this is attempting to fix? I'm not sure it will hold to user scrutiny.